utils/tests/test_fields.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+""" Tests utils/parsers.py """
+import re
+
+from utils.fields import BaseAPIField, ArrayField, EnumField
+from utils.constants import DataType
+from utils.fake_data import MilMoveProvider, MilMoveData
+from faker.generator import Generator
+
+
+class TestBaseAPIField:
+    """ Tests the BaseAPIField class and its methods """
+
+    @classmethod
+    def setup_class(cls):
+        """ Initialize the BaseAPIField that will be tested. """
+        cls.field1 = BaseAPIField(DataType.STREET_ADDRESS, name="streetAddress")
```

Comment:
Why `field1` when we have no other fields variables in this class? 👀

Reply:
Remnants from me trying to debug. Changed :)